docs/USAGE.rst

```diff
@@ -223,26 +234,30 @@ From the ``AvailableToProcess`` folder (or any sub folder) in the WSI DeID, clic
    :width: 200
    :alt: import button
 
-A background process starts that scans through the mounted import directory, and does the following:
+Clicking on the ``Import`` button will trigger a scan of the mounted import directory, and will do the following:
 
-- Each Excel file is parsed for a header row that has TokenID, ImageID, and InputFileName.
-- If there are any Excel files that do not have a header row, an error is generated and appears on the screen, and files are not imported.
-- If the same ScannedFileName is listed in multiple Excel files, the newest file is used by preference.
+- Each Metadata (Excel or CSV) file is parsed for a header row that has TokenID, ImageID, and InputFileName.
+- If there are any Metadata files that do not have a header row, an error is generated and appears on the screen, and files are not imported.
+- If the same ScannedFileName is listed in multiple Metadata files, the newest file is used by preference.
```

Comment:
`ScannedFileName` should be `InputFileName`.  I'd word this "the information in the newest DeID Upload Excel file is used by preference"